demonstrations/tutorial_qchem_external.py

```diff
@@ -0,0 +1,153 @@
+r"""
+
+Using PennyLane with PySCF and OpenFermion
+==========================================
+
+.. meta::
+    :property="og:description": Learn how to integrate external quantum chemistry libraries with PennyLane.
+    :property="og:image": https://pennylane.ai/qml/_images/thumbnail_tutorial_external_libs.svg
+
+.. related::
+    tutorial_quantum_chemistry Quantum chemistry with PennyLane
+    tutorial_vqe A brief overview of VQE
+    tutorial_givens_rotations Givens rotations for quantum chemistry
+    tutorial_adaptive_circuits Adaptive circuits for quantum chemistry
+
+*Author: Soran Jahangiri — Posted: 30 December 2022. Last updated: 30 December 2022.*
+
+The quantum chemistry module in PennyLane, :mod:`qml.qchem  <pennylane.qchem>`, provides built-in
+methods for computing molecular integrals, solving Hartree-Fock equations, and constructing
+`fully-differentiable <https://pennylane.ai/qml/demos/tutorial_differentiable_HF.html>`_ molecular
+Hamiltonians. PennyLane also allows using external libraries to perform these tasks and integrate
+the results within a desired workflow. In this tutorial, you will learn how to integrate PennyLane
+with `PySCF <https://github.com/sunqm/pyscf>`_ and
+`OpenFermion <https://github.com/quantumlib/OpenFermion>`_ to compute molecular integrals and
+construct molecular Hamiltonians.

```

Comment:
```suggestion
The quantum chemistry module in PennyLane, :mod:`qml.qchem  <pennylane.qchem>`, provides built-in
methods to compute molecular integrals, solve Hartree-Fock equations, and construct
`fully-differentiable <https://pennylane.ai/qml/demos/tutorial_differentiable_HF.html>`_ molecular
Hamiltonians. However, there are many other interesting and widely used quantum chemistry libraries out there. Instead of reinventing the wheel, PennyLane lets you to take advantage of various external resources and libraries to build upon existing research. In this demo we will show you how to integrate PennyLane with `PySCF <https://github.com/sunqm/pyscf>`_ and
`OpenFermion <https://github.com/quantumlib/OpenFermion>`_ to compute molecular integrals and
construct molecular Hamiltonians.
```
Changed verb forms to fit key term definitions better and integrated with Guillermo's suggestion.